plugins/woocommerce-admin/client/two-column-tasks/completed-header.tsx

```diff
@@ -0,0 +1,245 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { useEffect, useState } from '@wordpress/element';
+import { EllipsisMenu } from '@woocommerce/components';
+import { recordEvent } from '@woocommerce/tracks';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { OPTIONS_STORE_NAME, WCDataSelector, WEEK } from '@woocommerce/data';
+import { Button, Card, CardHeader } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import {
+	CustomerFeedbackModal,
+	CustomerFeedbackSimple,
+} from '@woocommerce/customer-effort-score';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './completed-header.scss';
+import HeaderImage from './completed-celebration-header.svg';
+
+type TaskListCompletedHeaderProps = {
+	hideTasks: () => void;
+	keepTasks: () => void;
+	enableCES: boolean;
```

Comment:
I think we should probably prefix most of our boolean variables with `is` or `has`.  Prefixing with a verb could imply that they are a function.

Reply:
coming very late on this, as I'm checking on a bug introduced in this PR, but I have naming thoughts.

I agree with @joshuatf , booleans should always be prefixed with `is` or `has` with very few exceptions. This should have been named `isCESEnabled`.

Reply:
I saw that you ended up changing it to `showCustomerEffortScore`.  That's better but `show` is still a verb, this could quite easily be interpreted as a callback function that returns a boolean.